<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OcctQtControl</class>
 <widget class="QDialog" name="OcctQtControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="6">
      <widget class="QToolButton" name="toolButton_zoom_bottom">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-bottom.svg</normaloff>:/view-bottom.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="13">
      <widget class="QToolButton" name="toolButton_zoom_min">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/zoom_min.svg</normaloff>:/zoom_min.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QToolButton" name="toolButton_zoom_left">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-left.svg</normaloff>:/view-left.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolButton_zoom_3d">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-iso.svg</normaloff>:/view-iso.svg</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>300</number>
       </property>
       <property name="autoRepeatInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QToolButton" name="toolButton_zoom_extends">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/zoom_extend.svg</normaloff>:/zoom_extend.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QToolButton" name="toolButton_zoom_right">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-right.svg</normaloff>:/view-right.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="toolButton_zoom_back">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-back.svg</normaloff>:/view-back.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <widget class="QToolButton" name="toolButton_zoom_plus">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/zoom_plus.svg</normaloff>:/zoom_plus.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2000</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="10">
      <widget class="QToolButton" name="toolButton_zoom_top">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-top.svg</normaloff>:/view-top.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_line">
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_circle">
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QToolButton" name="toolButton_zoom_fromt">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-front.svg</normaloff>:/view-front.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_arc_arc">
       <property name="text">
        <string>Arc-Arc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="5">
      <widget class="QLabel" name="label_val_mouse_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_val_mouse_y">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_mouse_y">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_val_mouse_z">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_mouse_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_mouse_z">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2500</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="libicon/icons.qrc"/>
 </resources>
 <connections/>
</ui>
